src/background.js

```diff
@@ -1,3 +1,111 @@
+// only update only the mainWindowIdOld and the mainWindowId
```

Comment:
Nit - use docbloc comments format and more descriptive comment
```
/**
 * Update the icons of the specified window ID to the "main window" icon" (and remove the existing icon)
 */
```

Reply:
Also - can you add a comment specifying why for something that looks like it updates the window, needs to set the icon for each individual tab? 

What happens if a new tab is added to one of these windows?

Reply:
- ok, I will do that next time I can work on it
- my intention (assuming all is working correctly) is that if you click it, it simply sets your "MainWindow" (the same as if you right-click, hit options, wait for it to load, then select the window from the drop down and hit save).  The difference is, this acts as a quick-toggle (as well as an easy-to-identify indicator).  So clicking it once sets the "MainWindow" to the window you just clicked on.  And clicking it a second time will un-set it, so now MainWindow is set to 0 (aka turned off).
- If a new tab is added to one of these windows it should behave the same as before, (get moved to the main window if applicable), however if the window the tab is added to is the main window, then the icon should be set to the X icon.
- Basically my intention is that, to the user, when the user clicks it on a certain window, no matter what tab you're on, the icon for that window will have the special X icon.  So it shouldn't really follow any tabs, the icon should be consistent per-window if that makes sense.
- if a tab gets drug away (detached) from that window to another window that's not main, it will technically lose the special icon, because it's no longer on the mainWindow.  But to the user, it will just appear to adopt the icon of the destination window.

Reply:
I believe I've added the comments you requested